doc/services/shell/index.rst

```diff
@@ -74,6 +74,28 @@ procedure:
   to the shell.
 
 
+Telnet Backend
+==============
+
+Enabling :kconfig:option:`CONFIG_SHELL_BACKEND_TELNET` will allow users to use telnet
+as a shell backend. Connecting to it can be done using PuTTY or any ``telnet`` client.
+For example:
+
+.. code-block:: none
+
+  telnet <ip address> <port>
+
+By default the telnet client won't handle telnet commands. Although command support
+can be enabled with :kconfig:option:`CONFIG_SHELL_TELNET_SUPPORT_COMMAND`. This will
+give the telnet client access to a very limited set of supported commands but still
+can be turned on if needed. One of the command option it supports is the ``ECHO``
+option. This will allow the client to be in character mode (character at a time),
+similar to a uart backend in that regard. This will make the client send a character
```

Comment:
Suggestion: I think `UART` should be capitalized.

Reply:
Good point